chart/values.yaml

```diff
@@ -127,3 +127,253 @@ owlsui:
   enabled: false
 
   fullnameOverride: owlsui
+
+# HAproxy (https://github.com/bitnami/charts/tree/master/bitnami/haproxy)
+haproxy:
```

Comment:
IMO we should default to 3 instances of HAProxy, wdyt?

Reply:
Sure, makes sense